src/stylesheets/panels/_sources.scss

```diff
@@ -20,6 +20,7 @@
 		& > .source {
 			border: none;
 			flex: 0 0 auto;
+			overflow: hidden;
```

Comment:
I did a pull request back to my master and this got into my dx9s-transitions (and durations) branch via older work prior to it. Hopefully this will merge and resolve and conflicts automagically. otherwise, not a big deal. Don't want to remove it hear as it would want to remove it from your master.